src/renderer/content.ts

```diff
@@ -1,4 +1,93 @@
-import { EditorId } from '../interfaces';
+import { USER_DATA_PATH } from './constants';
+import { EditorId, EditorValues } from '../interfaces';
+import { fancyImport } from '../utils/import';
+import { readFiddle } from '../utils/read-fiddle';
+
+import * as fetchType from 'node-fetch';
+import * as fsType from 'fs-extra';
+import * as path from 'path';
+import * as semver from 'semver';
+
+const decompress = require('decompress');
```

Comment:
Any reason we're using `require` instead of `import` here? It seems like `decompress` has type `any` in this case.

Reply:
Fixed 0dd47cc4a5334eec56376d50803935bc43d514f0